internal/netsync/manager.go

```diff
@@ -1845,6 +1847,27 @@ func (m *SyncManager) TicketPoolValue() (dcrutil.Amount, error) {
 	return m.cfg.Chain.TicketPoolValue()
 }
 
+// Run starts the sync manager and all other goroutines necessary for it to
+// function properly and blocks until the provided context is cancelled.
+func (m *SyncManager) Run(ctx context.Context) {
+	log.Trace("Starting sync manager")
+
+	// Start the event handler goroutine.
+	m.wg.Add(1)
+	go m.eventHandler(ctx)
+
+	// Shutdown the sync manager when the context is canceled.
```

Comment:
s / ... is canceled. / ... is cancelled.

Reply:
Nice catch.  I'll address this once some of the others are merge since it's a multi PR rebase.

Reply:
Updated.